packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -2436,4 +2443,56 @@ https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scope
       // },
     },
   };
+
+  private _getAdjustedGasLimit = async <
+    T extends ethers.Contract,
+    K extends keyof T['functions']
+  >(
+    contract: T,
+    method: K,
+    args: Parameters<T['functions'][K]>,
+    overrides: ethers.CallOverrides = {},
+    gasLimitAdjustment: ethers.BigNumber = GAS_LIMIT_ADJUSTMENT
+  ): Promise<ethers.BigNumber> => {
+    const gasLimit = await contract.estimateGas[method as string](
+      ...args,
+      overrides
+    );
+    return gasLimit.mul(gasLimitAdjustment);
```

Comment:
this multiplies the gas by 10, not by 10%.  going to make a PR to fix.

Reply:
ohh i think it actually multiplies it by 110?  

```
const GAS_LIMIT_INCREASE_PERCENTAGE = 10;
const GAS_LIMIT_ADJUSTMENT = ethers.BigNumber.from(100).add(
  GAS_LIMIT_INCREASE_PERCENTAGE
);
```

so GAS_LIMIT_ADJUSTMENT is 100 + 10 = 110.